source/conf.py

```diff
@@ -66,10 +68,16 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = [
     "_build",
+    "conftest.rst",
     "modules.rst",
     "setup.rst",
+    "prereise/hydro/legacy/capacity_factor.rst",
+    "prereise/renewables/solar/wind.rst",
```

Comment:
Exclude outdated documentation. Hydro profiles and solar profiles using capacity factors and WIND toolkit, respectively, are not used anymore